crates/sui-sdk/Cargo.toml

```diff
@@ -7,47 +7,46 @@ publish = false
 edition = "2021"
 
 [dependencies]
-anyhow = "1.0.64"
-async-trait = "0.1.61"
-clap = { version = "3.2.17", features = ["derive"] }
-colored = "2.0.0"
-jsonrpsee = { git="https://github.com/wlmyng/jsonrpsee.git", rev= "b1b300784795f6a64d0fcdf8f03081a9bc38bde8", features = ["http-client"] }
-serde = { version = "1.0.144", features = ["derive"] }
-serde_with = { version = "2.1.0", features = ["hex"] }
-serde_json = "1.0.88"
-futures-core = "0.3.21"
-futures = "0.3.23"
-tokio = { workspace = true }
-bcs = "0.1.4"
-thiserror = "1.0.37"
+anyhow.workspace = true
```

Comment:
I thought we generally wanted to extricate `sui-sdk` from the workspace and `workspace-hack` (or does not using `workspace-hack` not preclude using the `workspace` for dependencies?)

Reply:
Also cc @stefan-mysten for this discussion as you'll be looking into distributing `sui-sdk` on crates.io etc.

Reply:
The whole post of this PR is to make it very easy to continue having the sdk be a part of the workspace while eliminating the larger down sides of workspace-hack

Reply:
Awesome, thanks for clarifying!

Reply:
I'm happy if we want to do something different for the sdk as well, this was just me trying to be helpful towards that effort :)